content/docs/tina-cloud/connecting-site.md

```diff
@@ -3,35 +3,61 @@ title: Connecting the site
 next: '/docs/tina-cloud/faq'
 ---
 
-Once you've created a project within the **Tina Cloud**, the next step is to connect your site. Once connected, your project's editors will be able to save content directly to it's GitHub repository, entirely from within your site.
+Once you've created a project within the **Tina Cloud**, the next step is to connect your site. Once connected, your project's editors will be able to save content directly to its GitHub repository, entirely from within your site.
 
 ## Enabling Tina Cloud in TinaCMS
 
 In the [Contextual Editing doc](/docs/tinacms-context/), we showed you how the Tina context is setup on your site.
 
-To now enable Tina Cloud on your site, we just need to tweak a few properties to make the site editable:
+To use Tina Cloud, change the `apiURL` to:
 
-| property        | description                                                                                                                                                                           |
-| --------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `clientId`      | This identifier associates your site with a Tina Cloud project. You can find your clientId on the [Dashboard](https://app.tina.io/projects) by viewing your project's configuration.  |
-| `isLocalClient` | When this is false, TinaCMS uses Tina Cloud's hosted API instead of a local server to load and save content during editing.                                                           |
-| `branch`        | This is the branch in which we will load/save content to/from while in edit-mode. This can be a dynamic value if you'd like to implement more complex branching, or it can be static. |
+```
+https://content.tinajs.io/content/<myClientId>/github/<myBranch>
+```
+
+`<myClientId>` is the value from the Tina Cloud dashboard, and `<myBranch>` is the branch which you wish to communicate with.
 
-### Example
+We recommend storing this URL in an environment variable so you can switch between `localhost` and Tina Cloud easily. Note that it's safe for this variable to be public so if you're using Next, prefix it with `NEXT_PUBLIC_` for client-side access.
 
 ```tsx
 // pages/_app.js
 import TinaCMS from 'tinacms'
 
+const apiURL = process.env.NEXT_PUBLIC_TINA_API_URL
```

Comment:
Here you recommend using an environment variable, but later you recommend doing it inline with a dynamic branch. 

I think this may be simpler if we leave out the environment variable section (I think devs will identify on their own if they want to use environment variables).